packages/pluggableWidgets/switch-web/src/__tests__/Switch.spec.tsx

```diff
@@ -0,0 +1,273 @@
+import { actionValue, EditableValueBuilder, Alert, AlertProps } from "@mendix/piw-utils-internal";
+import { mount, ReactWrapper } from "enzyme";
+import { createElement } from "react";
+
+import SwitchComponent, { SwitchProps } from "../components/Switch";
+import { SwitchContainerProps } from "../../typings/SwitchProps";
+import { Switch } from "../Switch";
+
+describe("Switch", () => {
+    let switchWrapper: ReactWrapper<SwitchContainerProps, any>;
+    let switchComponent: ReactWrapper<SwitchProps, any>;
+    let switchComponentWrapper: ReactWrapper<any, any>;
+    let checkbox: ReactWrapper<any, any>;
+    let switchButtonWrapper: ReactWrapper<any, any>;
+    let switchButton: ReactWrapper<any, any>;
+    let alert: ReactWrapper<AlertProps, any>;
+    const createAndFindElements = (props: SwitchContainerProps): void => {
+        switchWrapper = mount(createElement(Switch, props));
+        switchComponent = switchWrapper.find(SwitchComponent);
+        switchComponentWrapper = switchComponent.find(".widget-switch");
+        checkbox = switchComponent.find(".widget-switch-checkbox");
+        switchButtonWrapper = switchComponent.find(".widget-switch-btn-wrapper");
+        switchButton = switchComponent.find(".widget-switch-btn");
+        alert = switchComponent.find(Alert);
+    };
+    const createProps = (props?: Partial<SwitchContainerProps>): SwitchContainerProps => {
+        const defaultProps: SwitchContainerProps = {
+            name: "switch",
+            class: "company-switch-widget",
+            style: { color: "red" },
+            tabIndex: 0,
+            id: "com.mendix.widgets.custom.switch1",
+            booleanAttribute: new EditableValueBuilder<boolean>().withValue(false).build(),
+            action: undefined
+        };
+
+        return { ...defaultProps, ...props };
+    };
+
+    it("with editable value should render the structure correctly", () => {
```

Comment:
You can omit the "should" and append an "s" to "render"

Reply:
Same for the other it's

Reply:
Done, thanks.

Reply:
@Keraito if you can please review for Isa

Reply:
Addressed in https://github.com/mendix/widgets-resources/commit/6ea8d2e2d8b3494126ca9ef25c32c9fad33091d5